src/dftbp/dftb/getenergies.F90

```diff
@@ -74,6 +75,10 @@ subroutine calcEnergies(env, sccCalc, tblite, qOrb, q0, chargePerShell, multipol
     !> Multipole moments
     type(TMultipole), intent(in) :: multipole
 
+    !> DFTB multipole moments
+    type(TMdftb), intent(inout), allocatable :: mdftb
```

Comment:
this could turned to `intent(in)` (I see no reason, why called routine must be `intent(inout)`)